### PR TITLE
Gulp branch

### PR DESCRIPTION
**Pull request by mi4l**

### CODE REVIEW

File: mike_skalandunas/lib/greet.js [hunk 1]
```diff
@@ -2,4 +2,4 @@
 
 module.exports = function(name) {
   return 'hello ' + name;
-}
+};
```
Comment (warmbowski) [inline mike_skalandunas/lib/greet.js:5]: jshint doing it's job, amiright? :)

File: mike_skalandunas/gulpfile.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+var gulp = require('gulp');
+var jshint = require('gulp-jshint');
+var mocha = require('gulp-mocha');
+var appFiles = ['index.js', 'lib/**/*.js', 'bin/**/*.js'];
+var testFiles = ['./test/**/*.js'];
+var watcher = gulp.watch('./**/*.js', ['jshint:test']);
+
+watcher.on('change', function(event) {
+  console.log('File ' + event.path + ' was ' + event.type + ', running tasks...');
```
Comment (warmbowski) [inline mike_skalandunas/gulpfile.js:11]: does this rerun tasks as well, or just log to console? I ask because I haven't used gulp yet.

Comment (mi4l) [inline mike_skalandunas/gulpfile.js:11]: It looks like it all works properly.  Whenever I make a change, I can see the jshint task start up.  I think I did Mocha correctly, but I'm waiting to hear Tyler talk about it.

Comment (warmbowski) [inline mike_skalandunas/gulpfile.js:11]: that's fine. I just havent seen that npm module. theres a gulp.watch already built into gulp

**Pull request merged**